lib/linguist/languages.yml

```diff
@@ -7104,8 +7103,10 @@ Visual Basic 6.0:
   type: programming
   color: "#2c6353"
   extensions:
+  - ".cls"
```

Comment:
The first extension should be the primary extension for the language. This is tricky in this case but as the file can contain no VB6 code, I don't think it should be first. Move it down one.

Reply:
Oh, I was going for alphabetical order 😅. In that case, it's a bit tricky indeed, but a `.cls` file is much more likely to include VB6 code than a `.ctl` or `.Dsr` file. In the case of the `.Dsr` file, I already mentionned it [here](https://github.com/github/linguist/pull/6124#issuecomment-1284650876). And for the `.ctl` file, it the same issue and could contain a lot of designer metadata and little to no VB6 ([example 1](https://github.com/Jucko13/KeyboardLights/blob/e5b17df9fd3460a44bb48911197327177937c0d7/Program/bColor.ctl) and [example 2](https://github.com/Cth1003/BonziBuddy/blob/4164bd069b9f59ed629282c307c588a419397d27/src/CalendarVB.ctl)).

In this regard, `.cls` is the primary extension for VB6 (along side `.bas`) since its only purpose is to contain VB code.